solr/core/src/java/org/apache/solr/update/CommitUpdateCommand.java

```diff
@@ -54,6 +57,7 @@ public String toString() {
             +",expungeDeletes="+expungeDeletes
             +",softCommit="+softCommit
             +",prepareCommit="+prepareCommit
+            +",commitData="+commitData
```

Comment:
instead please only do this if there is some data.  This will be rare.  I think this toString gets printed in logs and it's already rather verbose given some of these settings are more rare.  If you are feeling slightly ambitious, maybe only add the "true" & non-null items but not others.